src/main/java/com/example/onlinestore/user/UserRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.example.onlinestore.user;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface UserRepository extends JpaRepository<User, Long> {    
+        User findByUserName(String userName);
```

Comment:
You can format file automatically with `Ctrl`+`Alt`+`L`
```suggestion
    User findByUserName(String userName);
```

Reply:
noted :D